src/main/kotlin/com/terraformation/backend/species/db/SpeciesStore.kt

```diff
@@ -58,6 +61,27 @@ class SpeciesStore(
                 .toSet()
           }
 
+  private val usedInAccessions: Condition =
+      DSL.exists(
+          DSL.selectOne()
+              .from(ACCESSIONS)
+              .where(ACCESSIONS.SPECIES_ID.eq(SPECIES.ID))
+              .and(ACCESSIONS.facilities.ORGANIZATION_ID.eq(SPECIES.ORGANIZATION_ID)))
```

Comment:
You don't need the organization ID filter here or in the other conditions. Species are per-organization so if the species ID matches, the species must be in the right organization.

Reply:
I didn't account for the species ids being unique across orgs, didn't think about indices to be honest. I can add an index on species ID for those tables.

Reply:
Some of them might already have indexes (I only looked at accessions).

Reply:
Yes, batches has an index on species_id. Accessions and plantings don't, yet.

Reply:
Should I be adding these to `R__Indexes.sql` vs a new migration file?

Reply:
Plantings has an index as well, just accessions.